services/requester/cadence/evm_address_balance.cdc

```diff
@@ -1,8 +1,8 @@
 import EVM from 0xf8d6e0586b0a20c7
 
 access(all)
-fun main(addressBytes: [UInt8; 20]): UInt {
+fun main(addressBytes: [UInt8; 20]): UFix64 {
```

Comment:
I think we need to revert this change, as this is the old version of the script.

Reply:
Or is it compatible with the version of flow-go used in this PR?